ajax/libs/parsimmon/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "parsimmon",
+  "description": "A monadic LL(infinity) parser combinator library",
+  "filename": "src/parsimmon.min.js",
```

Comment:
Main file for the browser with the relative path to `basePath` in auto-update config:
`parsimmon.umd.min.js`